1 table.sql

```diff
@@ -0,0 +1,18 @@
+
+use musicrentalDB
+go
+drop table if exists dbo.MusicRental
+go
+create table dbo.MusicRental(
+    MusicRentalID int not null identity primary key,
+    CustomerFirstName varchar (100) not null constraint ck_MusicRental_customer_first_name_cannot_be_blank check(CustomerFirstName <> ''),   
+    CustomerLastName varchar (100) not null constraint ck_MusicRental_customer_last_name_cannot_be_blank check(CustomerLastName <> ''),
+    Instrument varchar(100) not null constraint ck_MusicRental_insrument_cannot_be_blank check (Instrument <> ''),
```

Comment:
Just missing the 't' in 'instrument' in the above constraint name

Reply:
sorry silly mistake